discgolfbot/pdga/pdga_cog.py

```diff
@@ -0,0 +1,196 @@
+                await interaction.send(embed=embed, content=f"{interaction.user.mention}:")
+
+    def get_www_pdga_com_user_data(
+            self,
+            pdga_player_number
+    ):
+        try:
+            pdga_player_scraper = PlayerScraper(pdga_number=f"{pdga_player_number}")
+
+            pdga_player_data = pdga_player_scraper.scrape()
+
+            if isinstance(pdga_player_data.player_name, str) and len(pdga_player_data.player_name) >= 3:
+                embed_title = "www.pdga.com - {}".format(pdga_player_data.player_name)
+            else:
+                embed_title = "www.pdga.com player information"
+                
+            embed = nextcord.Embed(title=embed_title, color=0x004899)
+            if isinstance(pdga_player_data.portrait_url, str) and re.match(pattern="^https{0,1}://", string=pdga_player_data.portrait_url):
+                embed.set_thumbnail(url=pdga_player_data.portrait_url)
+
+            embed_contents = "" 
+            desc_contents = ""
+            key_long = 17
+            value_long = 55
+            total = 62
+            pdga_player_data_dict = pdga_player_data.prettify()
+            for key in pdga_player_data_dict:
+                if None == pdga_player_data_dict[key]:
+                    continue
+                if key == 'Upcoming Events':
+                    continue
+                insert = f"{pdga_player_data_dict[key].lstrip().rstrip()}"
+                wordlength = len(f"{key}:{pdga_player_data_dict[key].lstrip().rstrip()}")
+                add_value_spaces = value_long - wordlength
+                add_key_spaces = key_long-len(key)
+                insert_key_spaces = ""
+                insert_value_spaces = ""
+                if wordlength < total:
+                    if len(key) > key_long:
+                        insert_key_spaces = " "*add_key_spaces
+                    if len(insert) < value_long:
+                        insert_value_spaces = " "*add_value_spaces
+                # embed_contents
+                desc_contents += f"\n`{key}{insert_key_spaces}:{insert_value_spaces}{insert}`"
+            embed.description = desc_contents #f"```elm\n{desc_contents}```"  # Player bio:\n\n
+
+
+            if 'Upcoming Events' in pdga_player_data_dict:
+                if len(pdga_player_data_dict["Upcoming Events"]) >= 1:
+                    embed.add_field(name="Upcoming events:", value=pdga_player_data_dict['Upcoming Events'], inline=True)
+                    #embed.description = f"Upcoming events: \n\n{pdga_player_data_dict[key]}"
+                    # FIXME: Validate upcoming events content length 
+                    pass
+        except:
+            if not 'embed' in locals():
+                embed_title = "Sorry, I failed you :("
+                embed = nextcord.Embed(title=embed_title, color=0x004899)
+        finally:
+            return embed
+        
+    def __get_user_by_nick__(self, interaction: Interaction, nick):
```

Comment:
Ser for meg at denne ikke er nødvendig så lenge man bruker typen nextcord.member i som parameter i slash-option.
Se f.eks;
@bag_slash_command.subcommand